0004-siren-prototype-accept-decline.adoc

```diff
@@ -0,0 +1,90 @@
+= Siren Prototype for comit-rs HTTP API - Accept/Decline actions
+Franck Royer <franck.royer@coblox.tech>
+:toc:
```

Comment:
:x: Can we have the `toc` after the initial metadata (author, date and tracking issue)?

Reply:
No we can't as far as I understand/tried. `:toc:` keyword only works directly after the title.

Reply:
can confirm.. my mistake.